edgexfoundry/edgex-compose
# fix: Remove :z and ,z from all named volumes

BREAKING CHANGE: This was previously add for CentOS, but is not causing issuse with move to `docker compose V2`. Users that require this for CentOS will need to customize the compose files appropraitly.

closes #329

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Run `make build` from compose builder.
Verify the generated compose files no longer have `bind` for `selinux: z`
